src/current/_includes/releases/v24.3/v24.3.0-beta.2.md

```diff
@@ -0,0 +1,38 @@
+## v24.3.0-beta.2
+
+Release Date: October 28, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-0-beta-2-sql-language-changes">SQL language changes</h3>
+
+- If a table is the destination of a logical replication stream, then only schema change statements that are deemed safe are allowed on the table. Safe statements are those that do not result in a rebuild of the primary index and do not create an index on a virtual computed column. [#133266][#133266]
+
+<h3 id="v24-3-0-beta-2-operational-changes">Operational changes</h3>
+
+- Two new metrics are available `sql.crud_query.count` and `sql.crud_query.started.count` which measure the number of [`INSERT`]({% link v24.3/insert.md %})/[`UPDATE`]({% link v24.3/update.md %})/[`DELETE`]({% link v24.3/delete.md %})/[`SELECT`]({% link v24.3/selection-queries.md %}) queries executed and started respectively. [#133198][#133198]
+- When creating a logical replication stream, any [user-defined types]({% link v24.3/create-type.md %}) in the source and destination are now checked for equivalency. This allows for creating a stream that handles user-defined types without needing to use the `WITH SKIP SCHEMA CHECK` option as long as the replication stream uses `mode = immediate`. [#133274][#133274]
+- Logical replication streams that reference tables with [user-defined types]({% link v24.3/create-type.md %}) can now be created with the `mode = immediate` option. [#133295][#133295]
+
+<h3 id="v24-3-0-beta-2-db-console-changes">DB Console changes</h3>
+
+- The [**SQL Statements**]({% link v24.3/ui-overview-dashboard.md %}#sql-statements) graph on the [**Overview**]({% link v24.3/ui-overview-dashboard.md %}) and [**SQL**]({% link v24.3/ui-sql-dashboard.md %}) dashboard pages in DB Console now show **Total Queries** as a general Queries Per Second (QPS) metric. [#133198][#133198]
+- Due to the inaccuracy of the **Range Count** column on the [**Databases** page]({% link v24.3/ui-databases-page.md %}) and the cost incurred to fetch the correct range count for every database in a cluster, this data will no longer be visible. This data is still available via a [`SHOW RANGES`]({% link v24.3/show-ranges.md %}) query. [#133267][#133267]
+
+<h3 id="v24-3-0-beta-2-bug-fixes">Bug fixes</h3>
+
+- Users with the `admin` role can now run [`ALTER DEFAULT PRIVILEGES FOR target_role ...`]({% link v24.3/alter-default-privileges.md %}) on any `target_role`. Previously, this could result in a privilege error, which is incorrect as `admin`s are allowed to perform any operation. [#133072][#133072]
+- [`REASSIGN OWNED BY current_owner_role ...`]({% link v24.3/reassign-owned.md %}) will now transfer ownership of the public schema. Previously, it would always skip over the `public` schema even if it was owned by the `current_owner_role`. [#133072][#133072]
```

Comment:
```suggestion
- [`REASSIGN OWNED BY current_owner_role ...`]({% link v24.3/reassign-owned.md %}) will now transfer ownership of the `public` schema. Previously, it would always skip over the `public` schema even if it was owned by the `current_owner_role`. [#133072][#133072]
```